src/anjaliP/Shop.java

```diff
@@ -0,0 +1,84 @@
+package anjaliP;
+
+public class Shop {
+	static int maggie = 50;
+	static int dosa = 43;
+	static int oilPouch = 39;
+	static int paniPuri = 43;
+	static int masala = 73;
+	
+	void customerItems(int maggie1 , int dosa1 , int oilPouch1 , int paniPuri1 , int masala1) 
+	{		
+		System.out.println("Customer Purchased :- "+"\n"+"--------------------------------------"
+				+"\n"+maggie1+" Maggie Packets"+"\n"+dosa1+" Dosa packets"+"\n"+oilPouch1+
+				" Oil Packets"+"\n"+paniPuri1+" Pani Puri packets "+"\n"+masala1+" Masala Packets. ");
+		System.out.println("----------------------Current status of Items in stock are----------------");
+		if (maggie==0)
+			System.out.println("Maggie is Out of Stock");
+		else if (maggie1>maggie)
+			System.out.println("Maggie is currently running out of Stock");
+		else maggie = maggie - maggie1;
+			System.out.println("Maggie Available in Stock : "+maggie);
```

Comment:
Here, If If we have 25 packet and customer asking for 26 packets, then you are displaying running out of Stock message! which is wrong, 
if this is the case then you should display remaining packets right, which you did, but one mistake you written in else if() condition so any single condition will executed.

think on this.